resources/electron/electron-plugin/src/server/api/notification.ts

```diff
@@ -26,22 +71,51 @@ router.post('/', (req, res) => {
 
     const notificationReference = reference ?? (Date.now() + '.' + Math.random().toString(36).slice(2, 9));
 
-    const notification = new Notification({
+    const usingLocalFile = isLocalFile(sound);
+
+    const createNotification = (opts: any) => {
+        try {
+            if (typeof (Notification as any) === 'function') {
+                return new (Notification as any)(opts);
+            }
+        } catch (e) {
+
+        }
+
+        return {
+            show: () => {},
+            on: (_: string, __: Function) => {},
+        };
+    };
+
+    const notification = createNotification({
         title,
         body,
         subtitle,
-        silent,
+        silent: usingLocalFile ? true : silent,
         icon,
         hasReply,
         timeoutType,
         replyPlaceholder,
-        sound,
+        sound: usingLocalFile ? undefined : sound,
         urgency,
         actions,
         closeButtonText,
         toastXml
     });
 
+    if (usingLocalFile && typeof sound === 'string') {
+        playSound(sound).catch((err) => {
+            notifyLaravel('events', {
+                event: '\\Native\\Desktop\\Events\\Notifications\\NotificationSoundFailed',
```

Comment:
I don't see this event anywhere in this PR. Is this even needed at this stage? If the sound fails, can't it just silently fail? (pun intended)

Reply:
I agree. If it errors somehow we should see it in the browser log. These events should be used for things devs can use on the Laravel side, not for errors